### ISSUE TITLE
[Feature(s)]

### ISSUE THREAD

**Issue by BioTechproject27** [Dev0Louis/FoggyBorder#1]
Hey, first off I wanna thank you for making this mod. It's odd to me that it took so long for anybody to improve the world border, it seemed so obvious?

Anyway, as to ideas that could improve the mod:
- The ability to give the fog a custom color (maybe also for expanding & contracting border).
- Have directional fog? It's odd that when approaching the world border, everything becomes covered in such thick fog, even if it's within the border. It should be less of a sphere, and more like a cubic wall. Vanilla rendering seems unfit here.

I also experienced some wird jittering(?)/flashing of the fog when approaching the border. The intensity depends on the players velocity it seems.

Anyway, yeah, pretty cool mod :3

**Comment by Dev0Louis** [comment]
Oh hey just seeing that now. 
> The ability to give the fog a custom color (maybe also for expanding & contracting border).

Oh yeah that would be really cool, I will do that as soon as I have time :)

> Have directional fog? It's odd that when approaching the world border, everything becomes covered in such thick fog, even if it's within the border. It should be less of a sphere, and more like a cubic wall. Vanilla rendering seems unfit here.

That is harder, I tried but it didn't work well. I will try again if I have an extended amount of time to work on it.

> I also experienced some wird jittering(?)/flashing of the fog when approaching the border. The intensity depends on the players velocity it seems.

Oh I will investigate that. I already have a guess what it could be.

Thank you for your Feedback :)

**Comment by Dev0Louis** [comment]
I fixed the jittering (I think?) try 1.1 of the Modrinth release :) https://modrinth.com/mod/foggy-border/version/1.1

**Comment by BioTechproject27** [comment]
Yup, works flawlessly :>

**Comment by FaultyFunctions** [comment]
I agree with the directional fog request. It's a bit odd that it surrounds you and prevents you from seeing anything.

**Comment by Dev0Louis** [comment]
I will revist the directional fog, not sure if I can do it though
